docs/testing.md

```diff
@@ -1,200 +1,171 @@
+KUDO includes a declarative integration testing harness for testing Operators, KUDO, and any other Kubernetes applications or controllers. Test cases are written as plain Kubernetes resources and can be run against a mocked control plane, locally in KIND, or any other Kubernetes cluster.
 
-## Table of Contents
+Whether you are developing an application, controller, operator, or deploying Kubernetes clusters the KUDO test harness helps you easily write portable end-to-end, integration, and conformance tests for Kubernetes without needing to write any code.
 
-* [Test harness usage](#test-harness-usage)
-   * [Run the Operator tests](#run-the-operator-tests)
-* [Writing test cases](#writing-test-cases)
-   * [Test case directory structure](#test-case-directory-structure)
-   * [Test steps](#test-steps)
-     * [Deleting resources](#deleting-resources)
-     * [Test assertions](#test-assertions)
-       * [Listing objects](#listing-objects)
-       * [Advanced test assertions](#advanced-test-assertions)
-* [Further Reading](#further-reading)
+* [Installation](#installation)
+* [Writing your first test](#writing-your-first-test)
+   * [Create a test case](#create-a-test-case)
+   * [Run the tests](#run-the-tests)
+   * [Write a second test step](#write-a-second-test-step)
+   * [Test suite configuration](#test-suite-configuration)
 
-## Test harness usage
+# Installation
 
-The Operator test suite is written and run by Operator developers and CI to test that Operators work correctly.
-
-First, clone the [Operators repository](https://github.com/kudobuilder/operators):
-
-```
-git clone https://github.com/kudobuilder/operators.git
-cd operators
-```
-
-Make sure that you have the latest version of kudoctl installed.
-
-#### Run the Operator tests
-
-To run the Operator test suite, run the following in the [Operators repository](https://github.com/kudobuilder/operators):
-
-```
-kubectl kudo test
-```
-
-To run against a production Kubernetes cluster, run:
+The test harness CLI is included in the KUDO CLI, to install we can install the CLI using [krew](https://github.com/kubernetes-sigs/krew):
 
 ```
-kubectl kudo test --start-kind=false
+krew install kudo
 ```
 
-Operator test suites are stored in the `tests` subdirectory of each [Operator](https://github.com/kudobuilder/operators/tree/master/repository), e.g.:
+You can now invoke the kudo test CLI:
 
 ```
-./repository/zookeeper/tests/
-./repository/mysql/tests/
+kubectl kudo test --help
 ```
 
-Every `Operator` and `OperatorVersion` is installed into the cluster by `make test` prior to running the test suite.
+See the [KUDO installation guide](/docs/cli#install) for alternative installation methods.
 
-## Writing test cases
+# Writing your first test
 
-To write a test case:
+Now that the kudo CLI is installed, we can write a test. The KUDO test CLI organizes tests into suites:
```

Comment:
maybe say somewhere that a suite has to always live in a separate directory?

Reply:
oh or the first level of folders is actually cases and one more level is suites, right? okay, that was not obvious to me on first read :) (just noting, in case you know how to improve on that)

Reply:
Hmm, so a test suite could actually be multiple directories since `testDirs` is a list, so I was avoiding talking about directories in the definitions section. I'm gonna think about this one some more.